lib/MountedBox.tsx

```diff
@@ -11,31 +11,37 @@ interface Props {
   className?: string;
 
   handlers: Record<`on${string}`, Func>;
+  children?: React.ReactNode;
 }
 
 /**
  * Mounts the given `UIBox` and attaches event handlers to it. Renders a
  * `loadingComponent` fallback until the mount is complete.
  */
 export function MountedBox(props: Props & { session: Talk.Session }) {
-  const { box, loadingComponent, className, handlers } = props;
+  const { box, loadingComponent, className, children, handlers } = props;
 
   const ref = useRef<HTMLDivElement>(null);
   const mounted = useMountBox(box, ref.current);
 
   const style = mounted ? props.style : { ...props.style, display: "none" };
 
   return (
-    <>
+    <BoxContext.Provider value={box}>
       {!mounted && (
         <div style={props.style} className={className}>
           {loadingComponent}
         </div>
       )}
 
       <div ref={ref} style={style} className={className} />
+      {children}
```

Comment:
maybe we should error out if there are children that aren't HTML Panels? Theoretically you could put anything in there, it's just a bit meaningless though, as it'll be identical to putting the children after the chatbox. If we error out then at least we don't accidentally allow something that breaks backward compat in some unspecified future.